### PR TITLE
Added docs to the page permissions routes

### PR DESCRIPTION
**Pull request by matanl490**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/software-catalog/page/page.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 1
+---
+
+# Page
+
+## What is a Page?
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:7]:
```suggestion
## What is a page?
```

File: docs/software-catalog/page/page.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+## Table operations
+
+Table operations are used to define the user’s view of the Port platform.
+
+:::tip
+We highly recommend using the table operations to provide a clean and accurate view of the platform for your developers.
+
+:::
+
+All table operations are available on the top bar of the table:
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:138]: Hasn't this been written already in line 70?

File: docs/software-catalog/page/page.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+![Table operations bar](../../../static/img/software-catalog/pages/TableOperationsBar.png)
+
+### Filter
+
+Filtering a table is done using the following menu:
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:144]:
```suggestion
YOu can filter a table by using the following menu:
```

File: docs/software-catalog/page/page.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+![Table filter menu marked](../../../static/img/software-catalog/pages/TableFilterMenu.png)
+
+You can define any filtering operator with a suitable value, in order to filter anything.
+
+You can filter according to one or many fields while setting the relation between each field filter: `And/Or`.
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:150]:
```suggestion
To filter, you can define any filtering operator with a suitable value.

You can filter one or more values while setting the relation between each field filter with a: `And/Or`.
```

File: docs/software-catalog/page/page.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+### Sort
+
+Sorting the table is done on the following menu:
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:154]:
```suggestion
You can sort the table on the following menu:
```

File: docs/software-catalog/page/page.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+
+You can sort by one or more fields of any kind.
+
+:::tip
+Sorting a specific column can be done by clicking on the column title.
+:::
+
+### Hide
+
+Hiding table columns is done on the following menu:
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:166]:
```suggestion
You can hide table columns on the following menu:
```

File: docs/software-catalog/page/page.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+
+You can decide whether each field is viewable to users or not.
+
+:::tip
+We highly recommend hiding irrelevant data from users, in order to provide them a clean work environment with few irrelevant distractions.
+:::
+
+### Group By
+
+Grouping by Entities is done on the following menu:
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:178]:
```suggestion
You can group by Entities on the following menu:
```

File: docs/software-catalog/page/page.md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+
+![Table group by menu marked](../../../static/img/software-catalog/pages/TableGroupByMenu.png)
+
+You can group results according to any field in the table.
+
+:::tip
+Group by is recommended when you want to create custom views for users, such as Microservices by Owners.
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:185]:
```suggestion
Group by is recommended when you want to create custom views for users, such as "microservices by owners".
```

File: docs/software-catalog/page/page.md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+Just create your `group by` setting, add additional viewing settings if needed, and [save a new page](#saving-new-pages) from the custom view.
+:::
+
+### Search
+
+We provide a free text search option on tables. Searching on a table is available on the left side of the top bar:
```
Comment (EllaFurman) [inline docs/software-catalog/page/page.md:192]:
```suggestion
We provide a free text search option on tables. To search on a table, click the left side of the top bar:
```

File: docs/software-catalog/page/tutorial.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+import TabItem from "@theme/TabItem"
+
+# Page Tutorial
+
+## Page permissions
+
+Pages have 4 regular CRUD permissions:
+
+- Create, Read, Update and Delete.
+- Currently only Read permission can be modified.
```
Comment (EllaFurman) [inline docs/software-catalog/page/tutorial.md:16]:
```suggestion
- Currently only read permission can be modified.
```

File: docs/software-catalog/page/tutorial.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+## Get page permissions
+
+- Any user can get the permissions of a specific page.
```
Comment (EllaFurman) [inline docs/software-catalog/page/tutorial.md:20]:
```suggestion
- Any user can get permissions for a specific page.
```

File: docs/software-catalog/page/tutorial.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+:::note
+Remember that an access token is needed to make API requests, refer back to [Getting an API token](../blueprint/tutorial.md#getting-an-api-token) if you need to generate a new one.
+:::
+
+Make an **HTTP GET** request to the URL: `https://api.getport.io/v1/pages/{Page_Identifier}/permissions`.
+
+The response will contain the roles and users that are allowed to read (view) the requested page.
+
+:::note
+Only Page Permissions of Software Catalog Pages can be requested.
```
Comment (EllaFurman) [inline docs/software-catalog/page/tutorial.md:33]:
```suggestion
Only page permissions of software catalog pages can be requested.
```

File: docs/software-catalog/page/tutorial.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+:::
+
+## Update page permissions
+
+- Only users with the admin role can update the permissions of a specific page;
```
Comment (EllaFurman) [inline docs/software-catalog/page/tutorial.md:38]:
```suggestion
- Only admin users can update permissions of a specific page;
```

File: docs/software-catalog/page/tutorial.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+{
+  "read": {
+    "roles": ["Admin", "Member"],
+    "users": []
+  }
+}
+```
+
+:::info
+When updating the permissions of a page, be sure to include the list of roles and users that need permissions in every request. Any role or user that does not appear in the request body will lose permissions to the page (this is effectively how you deny permissions to a page from a user)
```
Comment (EllaFurman) [inline docs/software-catalog/page/tutorial.md:61]:
```suggestion
When updating page permissions, be sure to include a list of roles and users that need permissions in every request. Any role or user that does not appear in the request body will lose permissions to the page (this is also how you deny permissions from a user)
```

**Review by EllaFurman — CHANGES_REQUESTED** [review]

File: docs/software-catalog/page/page.md (hunk 2 above)
Comment (MPTG94) [reply docs/software-catalog/page/page.md:138]: Line 70 refers to page operations, this line refers to table operations

**Pull request merged**